dotnet/src/Functions/Functions.OpenApi/DocumentLoader.cs

```diff
@@ -46,10 +46,12 @@ internal static async Task<string> LoadDocumentFromFilePathAsync(
 
         if (!File.Exists(filePath))
         {
-            throw new FileNotFoundException($"Invalid URI. The specified path '{filePath}' does not exist.");
+            var exception = new FileNotFoundException($"Invalid file path. The specified path '{filePath}' does not exist.");
+            logger.LogError(exception, "Invalid file path. The specified path '{FilePath}' does not exist.", filePath);
```

Comment:
nit: file path is surrounded by quotes here but not below and neither does uri above